crates/net/network/src/metrics.rs

```diff
@@ -310,11 +310,17 @@ impl DisconnectMetrics {
 #[derive(Metrics)]
 #[metrics(scope = "network")]
 pub struct EthRequestHandlerMetrics {
-    /// Number of received headers requests
-    pub(crate) received_headers_requests: Counter,
+    /// Number of GetBlockHeaders requests received
+    pub(crate) eth_requests_get_block_headers_received_total: Counter,
 
-    /// Number of received bodies requests
-    pub(crate) received_bodies_requests: Counter,
+    /// Number of GetReceipts requests received
+    pub(crate) eth_requests_get_receipts_received_total: Counter,
+
+    /// Number of GetBlockBodies requests received
+    pub(crate) eth_requests_get_block_bodies_received_total: Counter,
+
+    /// Number of GetNodeData requests received
+    pub(crate) eth_requests_get_node_data_received_total: Counter,
```

Comment:
```suggestion
    /// Number of GetReceipts requests received
    pub(crate) eth_receipt_requests_received: Counter,

    /// Number of GetBlockBodies requests received
    pub(crate) eth_bodies_requests_received: Counter,

    /// Number of GetNodeData requests received
    pub(crate) eth_node_data_requests_received: Counter,
```

Reply:
Sounds good with respect to the name changes. I have mostly accepted these changes in https://github.com/paradigmxyz/reth/pull/8319/commits/9e9abddc2577c7517f314c59276896bc26071090 with slight modification:

1. added `_total` to all these metrics to align with issue #8150 for consistency across the codebase
2. `eth_receipt_requests_received` -> `eth_receipts_requests_received_total` to align casing with the other metrics

let me know if this is good and thanks for the comments 👍

Reply:
makes sense!